client/python/tests/test_transaction.py

```diff
@@ -0,0 +1,123 @@
+import urllib3
+from mock import *
+
+from transaction import *
+from client_cache import *
+
+class TestTransaction(object):
+    def __init__(self):
+        self.tests = [
+            self.test_read,
+            self.test_write,
+            self.test_delete,
+            self.test_commit
+        ]
+
+    def test_read(self):
+        print "test_read",
+        server = "www.google.com"
+        cache = ClientCache(server)
+
+        # Mock the request method
+        headers = {
```

Comment:
Mocking HTTP is necessary for testing the ClientCache. I think it's a hard way to go about testing the transaction. Can you mock ClientCache instead?